tests/.suitespec.json

```diff
@@ -116,6 +116,7 @@
         "flask": [
             "ddtrace/contrib/flask/*",
             "ddtrace/contrib/flask_cache/*"
+            "ddtrace/contrib/flask_login/*"
```

Comment:
```suggestion
            "ddtrace/contrib/flask_cache/*",
            "ddtrace/contrib/flask_login/*"
```

Reply:
If you add the `flask_login` files to the existing `flask` suite, you shouldn't need to also add the [template config section[(https://github.com/DataDog/dd-trace-py/pull/7004/files#diff-8166c31d8ba62357b4a7cac4407d8744e60cf6b6ba32b09af939807db4c8f490R654).